src/rspamd.c

```diff
@@ -701,8 +701,12 @@ spawn_workers (struct rspamd_main *rspamd_main, struct ev_loop *ev_base)
 					spawn_worker_type (rspamd_main, ev_base, cf);
 				}
 				else {
-					msg_err_main ("cannot create listen socket for %s at %s",
-							g_quark_to_string (cf->type), cf->bind_conf->name);
+					if (cf->bind_conf == NULL)
```

Comment:
Could you please add `{` and `}` around all `if-else` operator branches - it is the common style for Rspamd code.